app/webpacker/components/Tickets/TicketWorkbenches/CompetitionResultActionerView/MiscActions.jsx

```diff
@@ -2,23 +2,28 @@ import React from 'react';
 import { Header, List, Segment } from 'semantic-ui-react';
 import AbortProcess from './MiscActions/AbortProcess';
 import EventsMergedData from './MiscActions/EventsMergedData';
+import WarningsAndMessageButton from './MiscActions/WarningsAndMessageButton';
 
-export default function MiscActions({ ticketDetails, updateStatus }) {
+export default function MiscActions({ ticketDetails }) {
   return (
     <Segment>
       <Header>Misc Actions</Header>
       <List>
         <List.Item>
           <AbortProcess
             ticketDetails={ticketDetails}
-            updateStatus={updateStatus}
```

Comment:
Just curious, why can this line go away and what does it have to do with the current PR?

Reply:
Hm actually not related. `AbortProcess` never needed `updateStatus` - I introduced `updateStatus` for this class in https://github.com/thewca/worldcubeassociation.org/pull/12162, but in the same PR mostly I've removed the dependency and forgot to remove from here.

In this PR, I wanted to remove `updateStatus` completely from `CompetitionResultActionerView`, and since I found it no longer used in `AbortProcess`, I did it in same PR. Should I move it to a separate PR?